third_party/bigframes_vendored/ibis/expr/operations/reductions.py

```diff
@@ -18,6 +20,18 @@ class ApproximateMultiQuantile(Filterable, Reduction):
     dtype = dt.Array(dt.float64)
 
 
-__all__ = [
-    "ApproximateMultiQuantile",
-]
+class ArrayAggregate(Filterable, Reduction):
```

Comment:
Worried that our expansion of Ibis reduction semantics to add order_by might make this fragile to current or future ibis tree manipulations, since they will not account for our ordering clause.

Reply:
@tswast Do you have any thoughts about it?

Reply:
Since we're using a totally new node, this seems safe enough to me, but we should make sure we're coordinating with upstream.